.gitconfig

```diff
@@ -0,0 +1,7 @@
+[core]
+longpaths = true
+
+# Support for long paths on windows:
+# https://docs.microsoft.com/en-us/windows/win32/fileio/maximum-file-path-limitation?tabs=powershell#enable-long-paths-in-windows-10-version-1607-and-later
+
```

Comment:
Let's be diverse and have macOs and Linux also onbaord, if possible?
What about `filemode = false`?